src/solver/misc/write-command-line.cpp

```diff
@@ -0,0 +1,13 @@
+#include <antares/logs.h>
+#include <sstream>
+
+void WriteCommandLineIntoLogs(int argc, char** argv)
```

Comment:
Note that we could also have a more "structured" output if we print `StudyLoadOptions` instead of raw command line arguments.

Reply:
(Indeed actually it's not enough, there is also `Settings`)